### PR TITLE
[Aichat] Add index to execution_status in AichatRequests table

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR adds an index to the `execution_status` column in `AichatRequests` table.

## Links
[jira](https://codedotorg.atlassian.net/browse/LABS-1031)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by sanchitmalhotra126 — APPROVED** [review]

File: dashboard/db/schema.rb [hunk 1]
```diff
@@ -1702,7 +1703,7 @@
     t.index ["storage_app_id"], name: "index_project_commits_on_storage_app_id"
   end
 
-  create_table "projects", id: :integer, charset: "utf8mb4", collation: "utf8mb4_general_ci", force: :cascade do |t|
+  create_table "projects", id: :integer, charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
```
Comment (bencodeorg) [inline dashboard/db/schema.rb]: Not sure we want this change?

**Review by bencodeorg — CHANGES_REQUESTED** [review]
I think there's a rogue schema edit in here

File: dashboard/db/schema.rb (hunk 1 above)
Comment (fisher-alice) [reply dashboard/db/schema.rb]: oh weird - thanks! Will revert this line.

Comment (fisher-alice) [reply dashboard/db/schema.rb]: This line has updated unexpectedly in the past locally after I run a new migration. Does it ever happen to you?

**Review by bencodeorg — APPROVED** [review]

File: dashboard/db/schema.rb (hunk 1 above)
Comment (bencodeorg) [reply dashboard/db/schema.rb]: mmm yes that sounds semi-familiar...quick google suggests what you've got might be the default for mysql8 now?

https://dev.mysql.com/blog-archive/mysql-8-0-collations-migrating-from-older-collations/

**Pull request merged**